docs/requirements/hardware-proposal.md

```diff
@@ -0,0 +1,98 @@
+# Hardware proposal
+
+Current document covers requirements in proposal real hardware solutions,
+which meet requirements gathered earlier.
+
+## Microchip ATTPM20P
+
+* Link to documentation: [mcp_attpm20p]:<https://ww1.microchip.com/downloads/en/DeviceDoc/ATTPM20P-Trusted-Platform-Module-TPM-2.0-SPI-Interface-Summary-Data-Sheet-DS40002082A.pdf>
```

Comment:
> The ATTPM20P is offered to OEM and ODM manufacturers as a turnkey solution, including the firmware
integrated on the chip. If custom firmware requirements are needed please contact Microchip Sales for
more information.

This is ready-to-use TPM module, with proprietary firmware, which is exactly what we are trying to avoid. However, it has some references that may be worth keeping around, like storage size. Maybe we should create another document for keeping track of existing TPMs?

Reply:
@macpijan worth considering.
Currently, we have only one chapter in this document, about device(s) not meeting our requirements (eg. due to lack of interface).

Reply:
We could also, what SoCs those read-to-use TPMs have.